microapps/MapViewCalloutApp/app/src/main/java/com/arcgismaps/toolkit/mapviewcalloutapp/screens/TapLocationScreen.kt

```diff
@@ -166,9 +229,11 @@ fun CalloutOptions(
                 textStyle = LocalTextStyle.current.copy(textAlign = TextAlign.End)
             )
         }
-
-        Button(onClick = viewModel::clearMapPoint) {
-            Text(text = "Clear Tap Location")
+        Spacer(modifier = Modifier.size(10.dp))
+        Button(
+            enabled = (mapPoint != null && calloutVisibility),
```

Comment:
we should not disable this when callout is not visible. 
```suggestion
            enabled = (mapPoint != null),
```

Reply:
The button is to "Clear Callout map point", in my opinion having this Button enabled when Callout is not visible doesn't seem right to me. This is the same boolean condition used to display the Callout.